src/app/debug/table/table.component.html

```diff
@@ -18,13 +18,13 @@
       <i class="fa fa-upload"></i>
       <input data-cy-debug="upload" #uploadFileTable (change)="uploadReports($event)" type="file" id="uploadFileTable">
     </button>
-    <app-button data-cy-debug="openSelected" [title]="'Open Selected Reports'" [text]="'Open Selected'" (click)="openSelected()"></app-button>
-    <app-button data-cy-debug="delete" [title]="'Delete'" [icon]="'fa fa-trash'" (click)="deleteSelected()"></app-button>
+    <app-button data-cy-debug="openSelected" title="Open Selected Reports" text="Open Selected" (click)="openSelected()"></app-button>
+    <app-button data-cy-debug="delete" title="Delete" icon="fa fa-trash" (click)="deleteSelected()"></app-button>
 
     <form (keydown.enter)="$event.preventDefault();" (keyup.enter)="changeTableLimit($event)">
       <div class='input-group'>
-          <input data-cy-debug='displayAmount' type='number' class='form-control input-field'
-                 value='{{tableSettings.displayAmount}}' (blur)='changeTableLimit($event)'>
+        <input data-cy-debug='displayAmount' type='number' class='form-control input-field'
```

Comment:
Ik zou dubbele quotes verwachten: `data-cy-debug="displayAmount"`. Dan kan je in Cypress testen refereren als: `cy.get('[data-cy-debug="displayAmount"]'`. Je hebt in src en in cypress dezelfde quotes om displayAmount heen.